src/tcgwars/logic/impl/gen3/PopSeries1.groovy

```diff
@@ -256,14 +273,22 @@ public enum PopSeries1 implements LogicCardInfo {
         pokeBody "Insomnia", {
           text "Murkrow can't be Asleep."
           delayedA {
+            after APPLY_SPECIAL_CONDITION, {
```

Comment:
This doesn't cover the case where an ability blocker is removed from play while murkrow is asleep. I think something like
`delayedA {
            before APPLY_SPECIAL_CONDITION,self, {
             if(ef.type == ASLEEP){
                prevent()
              }
            }
          }
          onActivate {
            clearSpecialCondition(self, SRC_ABILITY, [ASLEEP])
          }`

Reply:
```suggestion
            delayedA {
            before APPLY_SPECIAL_CONDITION,self, {
              if(ef.type == ASLEEP){
                prevent()
              }
            }
          }
          onActivate {
            clearSpecialCondition(self, SRC_ABILITY, [ASLEEP])
          }
```